modules/web/src/assets/css/_images.scss

```diff
@@ -570,10 +570,6 @@
   @include mixins.provider-image('/assets/images/clouds/kubeone.svg', 130px, 30px);
 }
 
-.km-os-image-centos {
```

Comment:
IMO we should keep the CentOS icon for now otherwise if someone has older cluster with CentOS then it will cause a visual problem. Or is there no way for a CentOS cluster to exist after 2.27?

Reply:
CentOS was not working before as well. So there should be no possibility of having a cluster running CentOS.